<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>350</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>나눔고딕</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>비밀번호 찾기</string>
  </property>
  <widget class="QPushButton" name="btnFindPw">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>찾기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPwtoHome">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>홈으로</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>231</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>정보 입력</string>
   </property>
   <widget class="QLabel" name="findPwstID">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>23</y>
      <width>60</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>학       번 :  </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPwstID">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>20</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly|Qt::ImhLatinOnly</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="findPwName">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>50</y>
      <width>60</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>이       름 : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPwName">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>47</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhFormattedNumbersOnly</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPwBirth">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>77</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="findPwBirth">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>80</y>
      <width>60</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>생년 월일 :</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblPw">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblfindPW">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblind">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
